app/src/test/java/io/crate/plugin/PluginLoaderTest.java

```diff
@@ -83,32 +77,6 @@ public void testPluginWithCrateSettings() throws Exception {
         assertThat(settings.get("setting.for.crate"), is("foo"));
     }
 
-    @Test
-    public void testLoadPluginRegisteringScalarFunction() throws Exception {
```

Comment:
We do not promote our plugin infrastructure anymore, our plugin example repository is archived.